### PR TITLE
chore: add benchmarking script, additional benchmarks

### PR DESCRIPTION
**Pull request by mweberxyz**
Implements `npm run benchmark` and adds additional benchmarks to help catch regressions/improvements

# Example usage
```
% npm run benchmark

> @fastify/view@8.2.0 benchmark
> node benchmark.js

express.js: 24266.67 req/s
fastify.js: 96245.34 req/s
fastify-twig.js: 86080 req/s
fastify-ejs-async.js: 104426.67 req/s
fastify-ejs-with-layout.js: 89322.67 req/s
```

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: benchmark.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const { readdirSync } = require('fs')
```
Comment (Uzlopak) [inline benchmark.js:1]:
```suggestion
'use strict'

const { readdirSync } = require('fs')
```

Comment (mweberxyz) [reply benchmark.js:3]: 👍

**Review by mcollina — APPROVED** [review]
lgtm

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mweberxyz** [comment]
Just pushed a light refactoring and added benchmarks for all supported engines - requesting re-review.

**Review by gurgunday — APPROVED** [review]

**Review by lilpoozie2005 — APPROVED** [review]

**Pull request merged**